### DISCUSSION TITLE
Do extra disks also trigger alarms?

### DISCUSSION THREAD

**Discussion by yurividal** [henrygd/beszel#184] [Support] [answered]
Hi. I have a system which has more than one disk. 
In my compose file, I have:
```
volumes:
    - /mnt/EXTERNAL/.beszel:/extra-filesystems/disk1:ro
environment:
     FILESYSTEM: /dev/sda2
````

This works fine, and i see the sdb usage in the graphs:

![image](https://github.com/user-attachments/assets/5995bd22-3eec-4c80-a9ae-3d87fef96b35)

My question is, **will this one also trigger an alert if over the threshold?** If so, it it controlled by the same threshold as the main disk?

**Comment by henrygd** [comment]
Currently (0.4.0) it does not. The alert is only for the partition set as your root.

I'm curious what people think regarding the implementation on this.

Should we have a single value that is applied to all disks, that's triggered when one disk goes above the value and cleared when all disks are under that value?

Or do you need more control to define specific values for different disks?

Same situation for temperature alerts. From a dev perspective it's a lot easier to add as a single value.

**Comment by yurividal** [comment]
thanks for the consideration. To me, having alerts of disk usage on my other disks is crucial. I use external mounts as most of my docker storage, and image storage. So, knowing if a disk is approaching its limit is a must. The ability to set individual thresholds per disk is probably a bonus - in my case i would probably just set them all to 90%, but i can see how this could be useful.

**Comment by henrygd** [comment, marked as answer]
The disk usage alert has been changed in 0.6.0 to check all disks.

Let me know if you run into any issues with it.

**Comment by sy6sy2** [comment]
Hi @henrygd,
Thanks for this nice tool!
Do you have the "ability to set individual thresholds per disk" in your roadmap?

**Reply by tafaust** [reply]
That would be awesome! I have the use case where I only want to receive notifications for my secondary disk but not for the primary.
